### DISCUSSION TITLE
NVIDIA 的 GTC 會議開場 I am AI 影片

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#20] [General]
2021年Keynote：[GTC 2021 Keynote with NVIDIA CEO Jensen Huang](https://youtu.be/eAn_oiZwUXA)
2021年I am AI影片：https://youtu.be/zNX1knTo2F4
2020年I am AI影片：https://youtu.be/e2_hsjpTi4w
2019年I am AI影片：https://youtu.be/GWL1HNHDSq4
2018年I am AI影片：https://youtu.be/GiZ7kyrwZGQ
2017年I am AI影片：https://youtu.be/SUNPrR4o5ZA

**Comment by HSU-YU-JEN** [comment]
之前有看到NVIDIA發布的影片，講述它整個建模過程以及如和模仿真實世界的物理定律，真的超厲害的!
影片網址:https://www.youtube.com/watch?v=1qhqZ9ECm70&t=33s
